File: templates/java/es/text/code/99beepers.java
/*
 * Archivo: LugarMuchosConos.java
 * -----------------------------
 * Lugares 42 conos usando un ciclo for
 */
import stanford.karel.*;
public class LugarMuchosConos extends Karel {
   public void run() {
      moverse();
      /* repetir ponerCono muchas veces */
      for(int i = 0; i < 42; i++) {
      	ponerCono();
      }
      moverse();
   } 
}

File: templates/python/es/text/code/99beepers.py
# Archivo: LugarMuchosConos.py
# -----------------------------
# Places 42 conos usando un ciclo for
from karel.stanfordkarel import *

def main():
   moverse()
   # repetir poner_cono muchas veces
   for i in range(42):
      poner_cono()
   moverse()
